app/src/main/java/com/lagradost/cloudstream3/extractors/Sendvid.kt

```diff
@@ -0,0 +1,32 @@
+package com.lagradost.cloudstream3.extractors
+
+import com.lagradost.cloudstream3.utils.*
+import com.lagradost.cloudstream3.app
+import com.lagradost.cloudstream3.utils.M3u8Helper.Companion.generateM3u8
+
+class SendvidHttps: Sendvid() {
+    override val mainUrl: String = "https://www.sendvid.com"
```

Comment:
This is not required because of:
https://github.com/recloudstream/cloudstream/blob/master/app/src/main/java/com/lagradost/cloudstream3/utils/ExtractorApi.kt#L94-L98
But good thinking :pray: